roles/openshift_node/tasks/systemd_units.yml

```diff
@@ -5,6 +5,14 @@
     dest: /usr/local/bin/openshift-node
     mode: 0500
 
+- name: Remove existing systemd service
+  file:
+    path: "/etc/systemd/system/{{ openshift_service_type }}-node.service"
+    state: absent
+  when:
+  - not openshift_is_bootstrapped
+  - openshift_is_atomic | bool
```

Comment:
The atomic command will place a new systemd service file in this location when the system container is install or updated.

This task will break idempotency of atomic hosts for upgrades.  We can't have this task here.  In fact, we really shouldn't need this task at all.